docs/UserGuide.md

```diff
@@ -252,6 +252,16 @@ Examples:
 * `sort o/name` returns the contact list sorted in alphabetical order.
 * `sort o/date` returns the contact list sorted in chronological order.
 
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+If sort is entered after executing find, a sorted found list will be displayed.
```

Comment:
Do you think we could include this as one of the examples? I know some of the other commands show examples of what happens when we execute the command after certain commands